modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/client_resource.ftl

```diff
@@ -199,7 +199,7 @@ public interface ${schemaName}Resource {
 			public HttpInvoker.HttpResponse ${javaMethodSignature.methodName}HttpResponse(${parameters}) throws Exception {
 				HttpInvoker httpInvoker = HttpInvoker.newHttpInvoker();
 
-				<#if freeMarkerTool.hasHTTPMethod(javaMethodSignature, "patch", "post", "put")>
+				<#if freeMarkerTool.hasHTTPMethod(javaMethodSignature, "delete", "patch", "post", "put")>
```

Comment:
Should we allow the "get" also here? (I think that in that case should be removing the if) As we said that the body in a GET method is also allowed

Reply:
Yeah, but we are not using it internally and it has more drawbacks that the rest of the methods (it creates URLs that are not bookmarkable) so I prefer to track usage this way.